sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/AssistantsAsyncClient.java

```diff
@@ -3751,4 +3752,62 @@ public Mono<PageableList<VectorStoreFile>> listVectorStoreFileBatchFiles(String
             .map(vectorStoreFileList -> PageableListAccessHelper.create(vectorStoreFileList.getData(),
                 vectorStoreFileList.getFirstId(), vectorStoreFileList.getLastId(), vectorStoreFileList.isHasMore()));
     }
+
+    /**
+     * Create a vector store file by attaching a file to a vector store.
+     *
+     * @param vectorStoreId The ID of the vector store for which to create a File.
+     * @param fileId A File ID that the vector store should use. Useful for tools like `file_search` that can access
+     * files.
+     * @param chunkingStrategy The chunking strategy used to chunk the file(s). If not set, will use the auto strategy.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return description of a file attached to a vector store on successful completion of {@link Mono}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<VectorStoreFile> createVectorStoreFile(String vectorStoreId, String fileId,
+        VectorStoreChunkingStrategyRequest chunkingStrategy) {
```

Comment:
Can we rename this to just `VectorStoreChunkingStrategy? The `Request` suffix is not necessary.

Reply:
We might, it is the naming passed from the TSP 
https://github.com/Azure/azure-rest-api-specs/pull/29785/files#diff-43c0c86e96266cce267ecea77d3e09da27480e924192aff658e22354e0c5f690R37